packages/react-query/src/usePrefetchQueries.tsx

```diff
@@ -0,0 +1,17 @@
+import { useQueryClient } from './QueryClientProvider'
+import type { FetchQueryOptions, QueryClient } from '@tanstack/query-core'
+
+export function usePrefetchQueries(
+  options: {
+    queries: ReadonlyArray<FetchQueryOptions<any, any, any, any>>
+  },
+  queryClient?: QueryClient,
+) {
```

Comment:
is `FetchQueryOptions<any, any, any, any>` really good enough here?

Reply:
I extracted out the rather through recursive `SuspenseQueriesOptions` recursive type reducer from `useSuspenseQueries` and modified it for using `FetchQueryOptions` in this case. Although were not gonna be using all that type information in this case at this point.